### ISSUE TITLE
There shouldn't be a PostTask flow event from ThreadProxy::ScheduledActionDrawAndSwap to MessageLoop::RunTask that handles next input event

### ISSUE THREAD

**Issue by yuhao** [google/trace-viewer#1045]
In the following trace: https://github.com/natduca/measurmt-traces/blob/master/mobile/googlemaps_ipc_newbinding.html

There is an outgoing PostTask flow event from ThreadProxy::ScheduledActionDrawAndSwap starting at 4,568.656 ms to a MessageLoop::RunTask event starting at 4,578.056 ms. As per discussion with Brian Anderson, this doesn't seem likely to happen on the C++ side. Is this a bug of the visualization? There are a couple of other occurrences in this trace.

My guess is that the arrow is pointing to a MessageLoop::RunTask that's not traced on the C++ side.

@natduca @dj2

**Comment by dj2** [comment]
All MessageLoop::RunTask's are traced on the c++ side. Can you look at the actual file and see how the id's match up? Is it just that it was bound to the wrong place?

**Comment by yuhao** [comment]
Seems like we always bind PostTask finish phase to the next slice. Is this how we implemented on the C++ side? In this case, the ts of the PostTask flow event is the same at the ts of MessageLoop::RunTask, and so we bind the flow event to it.

**Comment by natduca** [comment]
Is this a different issue than the rounding issue? I'm getting a little lost in the different bugs :)

**Comment by yuhao** [comment]
It doesn't look like a rounding issue. By looking at the raw trace file, it doesn't make sense to even have that flow event there. That's why I created a new issue for this.

**Comment by natduca** [comment]
Maybe we should switch messageloop binding point to parent?

**Comment by yuhao** [comment]
We will revisit this after #1079 is resolved. This might be solved by the new flow event API, or there is some behavior of Chrome we don't yet understand.
